### ISSUE TITLE
Possibility of setting Adam optimizer for TARS Classifier

### ISSUE THREAD

**Issue by FaezeGhorbanpour** [flairNLP/flair#2995]
Hi,

I appreciate your beneficial library. In this issue, I have a question regarding TARS Classifier and its optimizer.

I was applying few-shot learning to my own dataset using your library. In the optimization, I wanted to use different optimizers, such as Adam or AdamW, but whenever I used Adam (`torch.optim.Adam`), all the required metrics became zero! 

Would it be possible to use other optimizers for the TARS classifier? What configuration is needed if it is possible?

Thank you in advance.

**Comment by helpmefindaname** [comment]
Hi @FaezeGhorbanpour
No matter which model, you can always call `trainer.train(..., optimizer=torch.optim.Adam)`, or you can use `trainer.finetune(...)` which uses Adam per default.
For seeing more of the options, you can look at the [docstrings](https://github.com/flairNLP/flair/blob/master/flair/trainers/trainer.py#L129) of the train method.

**Comment by FaezeGhorbanpour** [comment]
Thank you for your response.

However, when I set the optimizer to `torch.optim.Adam`, the metrics became zero, and during the training, they did not change! Furthermore, when I tested using the function `trainer.finetune(...)`, I received an error indicating that the object "Trainer" does not have the "finetune"! 

If you could provide me with an example of how to use Adam in flair NLP models, particularly for few-shot models, I would really appreciate it.

**Comment by helpmefindaname** [comment]
Sorry for the inprecission the function is called [fine_tune](https://github.com/flairNLP/flair/blob/master/flair/trainers/trainer.py#L902) having metrics become zero usually indicates that the learning rate is too high (or too low). Did you account for adam needing a different lr than sgd?

An example of training a model with Transformers and Adam is [here](https://github.com/flairNLP/flair/blob/master/resources/docs/TUTORIAL_7_TRAINING_A_MODEL.md#training-a-named-entity-recognition-ner-model-with-transformers) It works the same when you replace the SequenceTagger with the TarsNer model.

**Comment by FaezeGhorbanpour** [comment]
Hi, 
Sorry, for late response. 

Yes, `fine_tune` was the correct function and I could use it, thank you. 

Still, whenever I set Adam as the optimizer, metrics got zero across a broad range of learning rates.
